packages/itwinui-react/src/core/Slider/Slider.tsx

```diff
@@ -490,6 +529,7 @@ export const Slider = React.forwardRef((props, ref) => {
           );
         })}
         <Track
+          {...sliderTrackProps}
```

Comment:
should go at the end. only `className` prop (if present) should go after it

Reply:
fixed